rocky/rocky/views/mixins.py

```diff
@@ -62,7 +62,10 @@ def get_observed_at(self) -> datetime:
 
         try:
             datetime_format = "%Y-%m-%d"
-            return convert_date_to_datetime(datetime.strptime(observed_at, datetime_format))
+            date_time = convert_date_to_datetime(datetime.strptime(observed_at, datetime_format))
+            if date_time.date() > datetime.now(timezone.utc).date():
+                messages.error(self.request, _("Beware: You have chosen for a date in the future."))
```

Comment:
If the data is still retrieved then this should maybe be a warning instead of an error.

Reply:
Ahhh yes yes good one!

Reply:
Just did:
![image](https://github.com/minvws/nl-kat-coordination/assets/30832687/28771cc0-8ae3-44b9-ba87-9d37290a2e26)